plugins/hls-fourmolu-plugin/src/Ide/Plugin/Fourmolu.hs

```diff
@@ -66,7 +66,7 @@ provider plId ideState typ contents fp fo = withIndefiniteProgress title Cancell
                 (exitCode, out, err) <-
                     readCreateProcessWithExitCode
                         ( proc "fourmolu" $
-                            ["-d"]
+                            ["-d", "--no-cabal"]
```

Comment:
Well, here's a reason for the "compile against the package" approach: we don't have a sane way of knowing whether or not to pass this flag, since it depends on which CLI version the user has installed!